doc/ref/plotting_options/styling.ipynb

```diff
@@ -0,0 +1,519 @@
+    "    title=\"Earthquakes Colored by Magnitude\"\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "1c34a309-d7c0-423e-adc7-9743f3d7a331",
+   "metadata": {},
+   "source": [
+    "(option-colormap)=\n",
+    "## `colormap`\n",
+    "\n",
+    "Alias for [`cmap`](option-cmap) above."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "3290b419-10a3-4d2b-ac99-0e50c644fbcc",
+   "metadata": {},
+   "source": [
+    "(option-color_key)=\n",
+    "## `color_key`\n",
+    "\n",
+    "The `color_key` option defines a categorical color mapping, primarily used with `datashade=True`. It maps distinct categories in your dataset to specific colors.\n",
+    "\n",
+    "It accepts:\n",
+    "- A list of colors (matched to unique categories in order)\n",
+    "- A dictionary mapping category values to colors\n",
+    "\n",
+    "If not provided, hvPlot uses a default categorical colormap. Only one of `cmap`, `colormap`, or `color_key` should be used at a time."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "c7c13149-05de-4fcd-96eb-6bd6e84dea06",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import hvplot.pandas  # noqa\n",
+    "import hvsampledata\n",
+    "\n",
+    "df = hvsampledata.earthquakes(\"pandas\")\n",
+    "\n",
+    "# Define explicit color mapping for depth_class\n",
+    "depth_colors = {\n",
+    "    'Shallow': '#33ff58',\n",
+    "    'Intermediate': '#ffc300',\n",
+    "    'Deep': '#f74a64'\n",
+    "}\n",
+    "\n",
+    "df.hvplot.scatter(\n",
+    "    x='lon',\n",
+    "    y='lat',\n",
+    "    c='depth_class',\n",
+    "    color_key=depth_colors,\n",
+    "    title=\"Earthquakes with Custom Color Key\"\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "b27eb1cc-3a6e-400b-816d-094a22bffac9",
+   "metadata": {},
+   "source": [
+    "(option-clim)=\n",
+    "## `clim`\n",
+    "\n",
+    "The `clim` option sets the lower and upper bounds of the color scale for continuous color mapping. It accepts a tuple like (min, max) and is useful when:\n",
+    "- You want consistent color scaling across multiple plots.\n",
+    "- You want to clip outliers or focus on a specific data range.\n",
+    "\n",
+    "If `clim` is not specified, the color scale is determined from the data (or percentiles if `robust=True`)."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "74d74cec-a37a-4be3-b8e2-9a607e20590d",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import hvplot.pandas  # noqa\n",
+    "import hvsampledata\n",
+    "\n",
+    "df = hvsampledata.earthquakes(\"pandas\")\n",
+    "\n",
+    "# Clamp the color scale to between 4.0 and 6.0\n",
+    "df.hvplot.scatter(\n",
+    "    x='lon',\n",
+    "    y='lat',\n",
+    "    c='mag',\n",
+    "    cmap='viridis',\n",
+    "    clim=(4.5, 6.5),\n",
+    "    title=\"Earthquakes with Clipped Magnitude Scale\"\n",
+    ") # It doesn't look like it worked"
```

Comment:
The `clim` keyword used in the example doesn't seem like it changed anything. I'm open to a better example that shows it working.

Reply:
Can you please open an issue?